packages/core/content-manager/admin/src/index.js

```diff
@@ -39,6 +40,8 @@ export default {
     // Hook that allows to mutate the CM's link pre-set filters
     app.createHook(MUTATE_COLLECTION_TYPES_LINKS);
     app.createHook(MUTATE_SINGLE_TYPES_LINKS);
+    // Hook that allows to mutate the CM's edit view layout
```

Comment:
WYT of putting this comment in the constant file? A la JSDOc so that when hovering the constant, we get details?